server/base-classes/transport.mdx

```diff
@@ -168,3 +168,26 @@ def add_event_handler(self, event_name: str, handler)
   Voice Activity Detection analyzer. You can set this to either
   `SileroVADAnalyzer()` or `WebRTCVADAnalyzer()`.
 </ParamField>
+
+#### Smart Turn Detection
+
+<ParamField path="end_of_turn_analyzer" type="Optional[BaseSmartTurn]" default="None">
+  Smart Turn Detection analyzer for advanced end-of-turn detection. You can set this to either 
+  `SmartTurnAnalyzer()` for remote inference or `LocalCoreMLSmartTurnAnalyzer()` for local inference.
```

Comment:
Again, I would remove `Smart Turn` wording here and mention `see for example [link to smart-turn]`.